schemas/log/log.json

```diff
@@ -0,0 +1,40 @@
+{
+    "title": "Log",
+    "description": "A device log message",
+    "type": "object",
+    "properties": {
+      "appId": {
+          "type": "string",
+          "const": "LOG"
+      },
+      "ts": {
+          "type": "number",
+          "description": "Unix timestamp given in milliseconds when the data was sampled"
+      },
+      "seq": {
+          "type": "number",
+          "description": "Sequence number of the message. This is a counter, used for ordering message if `ts` not available/used"
+      },
+      "domain": {
+          "type": "number",
+          "format": "int32",
+          "description": "Enumeration representing the source from which the message was sent from"
```

Comment:
```suggestion
          "description": "Enumeration representing the source from which the message was sent"
```

Reply:
Or "Enum value" (see above). Whatever you decide, just be consistent.